components/avatar.jsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * @fileoverview    React component for an avatar
+ * 
+ * @requires        NPM:react
+ * @requires        NPM:prop-types
+ */
+
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+/** 
+ * React component for an avatar icon
+ * 
+ * @param   {string} imgUrl - The avatar URL
+ * @param   {number} size - The height and width of avatar
+ */
+class Avatar extends Component {
+    render() {
+        let imgUrl = this.props.imgUrl ?  this.props.imgUrl : 'https://via.placeholder.com/32';
+        return (
+            <img src={imgUrl} height={this.props.size} width={this.props.size} className='rounded-img mr-3' />
+        );
+    }
+}
+
+Avatar.propTypes = {
+    imgUrl: PropTypes.string, // The avatar URL
+    size: PropTypes.number // The height and width of avatar
+};
```

Comment:
These sorts of things can be declared inside the class, which often makes it easier to read. For example:

```
export default class Avatar extends Component {
    static propTypes = {
        imgUrl: PropTypes.string,
        size: PropTypes.number,
    };
}
```

This also allows the last line (`export default`) to be merged into the class definition.

In the react-sdk we've also found that using TypeScript is often friendlier for development because it allows for a more strongly typed component - it also avoids the use of `PropTypes` entirely. Either works fine, it's just a personal preference thing.